Classes/DropDown/DropDownMenu.swift

```diff
@@ -8,17 +8,15 @@
 import SwiftUI
 
 struct DropDownMenu: View {
-    
     @EnvironmentObject var objGlobal : GlobalClass
     @State private var isOptionVisible: Bool = false
     @State var tag: Int = 0
-
+    @State var isSelected: Bool = false
+    @State var isSelectedIndex: Int = 0    
```

Comment:
[Comments]:
1. You don't need both isSelected and isSelectedIndex.
2. DropDownMenu doesn't need to know if it's for rooms or children selection. it can be very generic that takes any value.